File: JavaScript/two-sum-ii-input-array-is-sorted.js
var twoSum = function(numbers, target) {
  for (let low = 0, high = numbers.length - 1; low < high;) {
    if (numbers[low] + numbers[high] > target) {
      high--;
    } else if (numbers[low] + numbers[high] < target) {
      low++;
    } else {
      return [low + 1, high + 1];
    }
  }
};

File: Python/two-sum-ii-input-array-is-sorted.py
from ast import List

def twoSum(self, numbers: List[int], target: int) -> List[int]:
  low, high = 0, len(numbers) - 1
  while low < high:
    if numbers[low] + numbers[high] > target:
      high -= 1
    elif numbers[low] + numbers[high] < target:
      low += 1
    else:
      return low + 1, high + 1
